kura/org.eclipse.kura.wire.h2db.component.provider/OSGI-INF/metatype/org.eclipse.kura.wire.H2DbWireRecordStore.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2017 Eurotech and/or its affiliates and others
+
+     All rights reserved. This program and the accompanying materials
+     are made available under the terms of the Eclipse Public License v1.0
+     which accompanies this distribution, and is available at
+     http://www.eclipse.org/legal/epl-v10.html
+     
+     Contributors:
+      Eurotech
+      Amit Kumar Mondal
+-->
+<MetaData xmlns="http://www.osgi.org/xmlns/metatype/v1.2.0" localization="en_us">
+    <OCD id="org.eclipse.kura.wire.H2DbWireRecordStore"
+         name="H2 DB Store" 
+         description="A wire component that provides storage for Wire Records produced by connected WireEmitters.">
```

Comment:
"A wire component that provides storage for Wire Records produced by connected WireEmitters. The storage will be performed in the selected H2 Database instance."